src/registrar/views/index.py

```diff
@@ -1,9 +1,11 @@
 from django.db.models import F
 from django.shortcuts import render
+from django.contrib.auth.decorators import login_required
 
 from registrar.models import DomainApplication
 
 
+@login_required
 def index(request):
     """This page is available to anyone without logging in."""
```

Comment:
**suggestion**: We should change this docstring to accurately match the new decorator.